README.md

```diff
@@ -154,6 +156,8 @@ These are the years-of-introduction of various frameworks and library collection
 
 - Symfony 3.0.0: <http://symfony.com/blog/symfony-3-0-0-released>
 
+- Simple MVC Framework 2.1: <https://packagist.org/packages/simple-mvc-framework/v2#v2.1>
```

Comment:
Hi @sergeyklay . The [framework ](https://packagist.org/packages/tomvlk/simplemvcframework)don't have the major release version on packagist.org.

It has only 2.1 and 2.2